lib/src/main/java/graphql/nadel/engine/blueprint/NadelExecutionBlueprintFactory.kt

```diff
@@ -218,6 +219,7 @@ private class Factory(
         val hydrationActorService = services.single { it.name == hydration.serviceName }
         val queryPathToActorField = hydration.pathToActorField
         val actorFieldDef = engineSchema.queryType.getFieldAt(queryPathToActorField)!!
+        val actorFieldContainer = engineSchema.queryType.getFieldContainerAt(queryPathToActorField)!!
```

Comment:
This is the key nit of code - we capture both the "actor field def AND the container of that actor field"

We can later use it to make the hydration details object